cranelift/codegen/src/machinst/abi_impl.rs

```diff
@@ -94,34 +94,6 @@
 //!
 //!   (low address)
 //! ```
-//!
-//! # Multi-value Returns
-//!
-//! Note that we support multi-value returns in two ways. First, we allow for
-//! multiple return-value registers. Second, if teh appropriate flag is set, we
-//! support the SpiderMonkey Wasm ABI.  For details of the multi-value return
```

Comment:
Maybe keep this comment, but replace the description of spidermonkey with one of wasmtime's call conv?

Reply:
Updated, thanks.